doc/plugin_tutorial/tuto2/src/persistent_counter.ml

```diff
@@ -0,0 +1,57 @@
+(*
+ * This file defines our persistent counter, which we use in the
+ * CountPersistent command.
+ *)
+
+(*
+ * As before, we want to override references:
+ *)
+open Summary.Local
```

Comment:
Why Local? (here and for the other counter too)

Reply:
Not sure, it's the one that was used in the old version of the tutorial. Should it not be used for either?

Reply:
I'm not sure what Local is for, in Coq itself it's only used by the ltac profiler.
Using regular Summary means the ref is a regular ref so at least that part is more convenient ;)

Reply:
OK, I'll change them to use Summary and update this tomorrow as long as it doesn't change behavior.